### DISCUSSION TITLE
Why power outage causing Zigbee hub to change its MAC address

### DISCUSSION THREAD

**Discussion by martinikolaev** [arendst/Tasmota#11779] [Q&A] [answered]
Hello,

I am observing MAC address change on power outage which is breaking my IP Binding
Network setup - pfsense box -> Unifi NanoHD AP -> Tasmota
Tasmota version: 9.3.1.2

![image](https://user-images.githubusercontent.com/41118170/115143290-19c2f600-a04f-11eb-8a60-6114f8d20135.png)

Have anyone observed this before?

**Comment by barbudor** [comment, marked as answer]
MAC address are normally burned down inside the ESP chip and shouldn't change
The top one is identified as Espressif
But the 2nd one seems bogus and not registered
Very strange

Which one is the original ? The top one ?
What happens if you power cycle again ?

**Reply by martinikolaev** [reply]
The original is the top one... 
When I powercycle it, it does changes it but not every time, like once a week or so on average.
I can see it changed from the UI -> Information menu, its not just the router. 
I was hoping to check with you guys if there are some logs or something to check the source of the change ;/

**Reply by s-hadinger** [reply]
We do nothing with regards to Mac address in Tasmota. I've never seen that behavior

**Reply by martinikolaev** [reply]
Happened once again

:(
On the DHCP from pfSense side I have DHCP Option 43, which is for my unifi devices.

![image](https://user-images.githubusercontent.com/41118170/115147939-f8213900-a065-11eb-8097-85d5a3e5a274.png)

**Reply by barbudor** [reply]
When it changes does to revert to the original ? Does it alternates always between both ?
I would suspect a faulty hardware
We can see that the last 3 digits remains the same
There is an API on ESP8266 to change the MAC address but Tasmota doesn't use it

**Reply by martinikolaev** [reply]
I havent noticed if it go back and forth these, since I just update my static IP record in the DHCP.

**Reply by martinikolaev** [reply]
Is it possible the bridge to have different MAC addresses on different APs ?

**Reply by sfromis** [reply]
The ESP8266 has *one* MAC address.
